zanata-war/src/main/java/org/zanata/webtrans/client/util/DateUtil.java

```diff
@@ -53,8 +44,7 @@ public static String formatTime(Date date) {
      */
     public static String formatLongDateTime(Date date) {
         if (date != null) {
-            return DateTimeFormat.getFormat(DATE_TIME_LONG_PATTERN)
-                    .format(date);
+            return dtfLong.format(date);
```

Comment:
The old version couldn't be tested without using GWTTestCase, which we won't work with TestNG.  GWT code all runs in a single Javascript thread anyway.